  void throwExtendsExtendsCustomException() throws ExtendsExtendsCustomException {
    throw new ExtendsExtendsCustomException();
  }

